sunpy/coordinates/tests/test_sun.py

```diff
@@ -203,12 +203,12 @@ def test_B0_jpl_horizons():
                   '2013-04-01': -6.543092,
                   '2013-05-01': -4.168016,
                   '2013-06-01': -0.663804,
-                  '2013-07-01':  2.873030,
```

Comment:
I propose we revert these, I think these columns of numbers were probably designed like this by @ayshih 

can we disable a single check per file?

Reply:
We could. Was it to make the file easy to read down?

Reply:
yeah, I assume so, as these are copied out of other references, it's presumably important to be able to read them quickly.

Reply:
I have added a commit that might cause an uproar.